applications/task-manager/main.cpp

```diff
@@ -113,8 +134,14 @@ int main(int argc, char **argv)
     ram_icon_and_text->layout = HFLOW(4);
     icon_panel_create(ram_icon_and_text, Icon::get("chip"));
     label_create(ram_icon_and_text, "Memory");
-
-    window->ram_timer = timer_create(window->ram_graph, 500, (TimerCallback)widget_ram_update);
+    
+    window->ram_usage = label_create(window->ram_graph, "Usage: nil Mio");
+    window->ram_avaliable = label_create(window->ram_graph, "Avaliable: nil Mio");
+    window->ram_greedy = label_create(window->ram_graph, "Most greedy: the compositor eats a lot");
```

Comment:
lol

Reply:
I will add the rest of the features, It won't take long

Reply:
it's taking a little longer than expected. I have to fix a bug I caused while working on it that made the task manager not work

Reply:
I finally got it to work, it was one tiny mistake. changed `list[i]` to `i` 😒